recommender/tasks.py

```diff
@@ -51,3 +56,31 @@ def add_user_action(user_action_raw: dict):
     """
     user_action = UserAction.parse_obj(user_action_raw)
     Deserializer.deserialize_user_action(user_action).save()
+
+
+@celery.task
+def send_recommendation_to_databus(context: dict, recommendation_response: dict):
+    host = current_app.config["RS_DATABUS_HOST"]
```

Comment:
Could you exclude those env variables? For example at the top of this file.

Reply:
do you mean extract?